### PR TITLE
Updated DM Profile Links for health-cards API

### PR DESCRIPTION
**Pull request by stefanioachim**
Description
Update broken links (DM Profiles) on fhir.cerner.com in the health-cards section 

PR Checklist

Health Cards documentation page with links to broken pages (e.g. VaccinationCredentialPatientDM)

Screen Shot 2022-08-16 at 11 03 51

The 404 page displayed for VaccinationCredentialPatientDM broken link :

Screen Shot 2022-08-16 at 11 05 41

----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

**Review by lstefan4 — APPROVED** [review]

File: content/millennium/r4/foundation/other/health-cards.md [hunk 1]
```diff
@@ -43,11 +43,11 @@ Represents the patient and the clinical data related to a vaccination
...
+* A single Patient resource that follows the [VaccinationCredentialPatientDM](https://build.fhir.org/ig/HL7/fhir-shc-vaccination-ig/StructureDefinition-shc-patient-us-resource-examples.html) profile with the following fields, if valued:
   * [Patient official name](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-vaccination-credential-patient-definitions.html#Patient.name){:target="_blank"}
   * [Date of Birth](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-vaccination-credential-patient-definitions.html#Patient.birthDate){:target="_blank"}
 
-* One or more Immunization resources that follow the [VaccinationCredentialImmunizationDM](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-vaccination-credential-immunization-dm.html) profile with the following fields, if valued:
+* One or more Immunization resources that follow the [VaccinationCredentialImmunizationDM](https://build.fhir.org/ig/HL7/fhir-shc-vaccination-ig/StructureDefinition-shc-vaccination-dm.html) profile with the following fields, if valued:
```
Comment (learclite) [inline content/millennium/r4/foundation/other/health-cards.md:50]: Just checking, is this link correct here? I'm not sure whether Alec pasted the same link for this and VaccinationCredentialPatientDM by mistake and if they should be different or not.

**Review by lstefan4 — APPROVED** [review]

File: content/millennium/r4/foundation/other/health-cards.md (hunk 1 above)
Comment (stefanioachim) [reply content/millennium/r4/foundation/other/health-cards.md:50]: > Just checking, is this link correct here? I'm not sure whether in the Jira comment Alec pasted the same link for this and VaccinationCredentialPatientDM by mistake and if they should be different or not.

Hi Florian, the new links are not the same : 
- For VaccinationCredentialImmunizationDM : https://build.fhir.org/ig/HL7/fhir-shc-vaccination-ig/StructureDefinition-shc-vaccination-dm.html
- For VaccinationCredentialPatientDM            : https://build.fhir.org/ig/HL7/fhir-shc-vaccination-ig/StructureDefinition-shc-patient-us-resource-examples.html

Comment (fpivonka) [reply content/millennium/r4/foundation/other/health-cards.md:50]: > > Just checking, is this link correct here? I'm not sure whether in the Jira comment Alec pasted the same link for this and VaccinationCredentialPatientDM by mistake and if they should be different or not.
> 
> Hi Florian, the new links are not the same :
> 
>     * For VaccinationCredentialImmunizationDM : [build.fhir.org/ig/HL7/fhir-shc-vaccination-ig/StructureDefinition-shc-vaccination-dm.html](https://build.fhir.org/ig/HL7/fhir-shc-vaccination-ig/StructureDefinition-shc-vaccination-dm.html)
> 
>     * For VaccinationCredentialPatientDM            : [build.fhir.org/ig/HL7/fhir-shc-vaccination-ig/StructureDefinition-shc-patient-us-resource-examples.html](https://build.fhir.org/ig/HL7/fhir-shc-vaccination-ig/StructureDefinition-shc-patient-us-resource-examples.html)

I think you should probably not use the 'build' version as the links will keep breaking. I think we should probably use the links from 2021Sep guide http://hl7.org/fhir/uv/shc-vaccination/2021Sep/StructureDefinition-shc-vaccination-dm.html and http://hl7.org/fhir/uv/shc-vaccination/2021Sep/StructureDefinition-shc-patient-us-dm.html

**Review by fpivonka — COMMENTED** [review]
I think we need to look at all the links - there are still a lot of broken links in the page for fhir.cerner.com and make sure they are updated using the correct links from a published guide from Sept 2021 http://hl7.org/fhir/uv/shc-vaccination/2021Sep/index.html as the existing links from the build repository will continue to break as the specification is continuously updated

**Comment by stefanioachim** [comment]
As per @fpivonka recommendation in the discussion with @lstefan4 I modified the "build" links to "latest published" links so that the content referenced by the links won't change every time a new build is made

**Review by lstefan4 — APPROVED** [review]

File: content/millennium/r4/foundation/other/health-cards.md [hunk 2]
```diff
@@ -43,11 +43,11 @@ Represents the patient and the clinical data related to a vaccination
 
 Contains the following resources:
 
-* A single Patient resource that follows the [VaccinationCredentialPatientDM](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-vaccination-credential-patient-dm.html#tab-ms) profile with the following fields, if valued:
+* A single Patient resource that follows the [VaccinationCredentialPatientDM](http://hl7.org/fhir/uv/shc-vaccination/2021Sep/) profile with the following fields, if valued:
```
Comment (fpivonka) [inline content/millennium/r4/foundation/other/health-cards.md:46]: So, this is better that you are linking to the 2021Sep version however, the link we are replacing was pointing to a specific profile for vaccine-credential-patient-dm which would be this page in the 2021Sep version http://hl7.org/fhir/uv/shc-vaccination/2021Sep/StructureDefinition-shc-patient-general-dm.html

File: content/millennium/r4/foundation/other/health-cards.md [hunk 3]
```diff
@@ -43,11 +43,11 @@ Represents the patient and the clinical data related to a vaccination
...
   * [Patient official name](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-vaccination-credential-patient-definitions.html#Patient.name){:target="_blank"}
   * [Date of Birth](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-vaccination-credential-patient-definitions.html#Patient.birthDate){:target="_blank"}
 
-* One or more Immunization resources that follow the [VaccinationCredentialImmunizationDM](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-vaccination-credential-immunization-dm.html) profile with the following fields, if valued:
+* One or more Immunization resources that follow the [VaccinationCredentialImmunizationDM](http://hl7.org/fhir/uv/shc-vaccination/2021Sep/) profile with the following fields, if valued:
```
Comment (fpivonka) [inline content/millennium/r4/foundation/other/health-cards.md:50]: same here - think we need to link to the specific immunization profile http://hl7.org/fhir/uv/shc-vaccination/2021Sep/StructureDefinition-shc-vaccination-bundle-dm.html

File: content/millennium/r4/foundation/other/health-cards.md [hunk 4]
```diff
@@ -70,11 +70,11 @@ Represents the patient and the clinical data related to a laboratory test result
...
+* A single Patient resource that follows the [VaccinationCredentialPatientDM](http://hl7.org/fhir/uv/shc-vaccination/2021Sep/) profile with the following fields, if valued:
   * [Patient official name](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-vaccination-credential-patient-definitions.html#Patient.name){:target="_blank"}
   * [Date of Birth](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-vaccination-credential-patient-definitions.html#Patient.birthDate){:target="_blank"}
 
-* One or more Observation resources that follow the [InfectiousDiseaseLaboratoryResultObservationDM](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-infectious-disease-laboratory-result-observation-dm.html) profile with the following fields, if valued:
+* One or more Observation resources that follow the [InfectiousDiseaseLaboratoryResultObservationDM](http://hl7.org/fhir/uv/shc-vaccination/2021Sep/) profile with the following fields, if valued:
```
Comment (fpivonka) [inline content/millennium/r4/foundation/other/health-cards.md:77]: should be to the specific lab results data minimization profile http://hl7.org/fhir/uv/shc-vaccination/2021Sep/StructureDefinition-shc-infectious-disease-laboratory-result-observation-dm.html

File: content/millennium/r4/foundation/other/health-cards.md (hunk 2 above)
Comment (stefanioachim) [reply content/millennium/r4/foundation/other/health-cards.md]: > So, this is better that you are linking to the 2021Sep version however, the link we are replacing was pointing to a specific profile for vaccine-credential-patient-dm which would be this page in the 2021Sep version http://hl7.org/fhir/uv/shc-vaccination/2021Sep/StructureDefinition-shc-patient-general-dm.html I think we need to link to the specific profile where warranted.

Fixed in commit https://github.com/cerner/fhir.cerner.com/pull/874/commits/6891a9e92a9faf1db82a825354e912180f12c6af

File: content/millennium/r4/foundation/other/health-cards.md (hunk 3 above)
Comment (stefanioachim) [reply content/millennium/r4/foundation/other/health-cards.md]: > same here - think we need to link to the specific immunization profile http://hl7.org/fhir/uv/shc-vaccination/2021Sep/StructureDefinition-shc-vaccination-dm.html

Fixed in commit https://github.com/cerner/fhir.cerner.com/pull/874/commits/6891a9e92a9faf1db82a825354e912180f12c6af

File: content/millennium/r4/foundation/other/health-cards.md [hunk 5]
```diff
@@ -70,11 +70,11 @@ Represents the patient and the clinical data related to a laboratory test result
 
 Contains the following resources:
 
-* A single Patient resource that follows the [VaccinationCredentialPatientDM](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-vaccination-credential-patient-dm.html#tab-ms) profile with the following fields, if valued:
+* A single Patient resource that follows the [VaccinationCredentialPatientDM](http://hl7.org/fhir/uv/shc-vaccination/2021Sep/) profile with the following fields, if valued:
```
[the remark this answers is not in the document]
Comment (stefanioachim) [reply content/millennium/r4/foundation/other/health-cards.md]: > same comment as on line 46 - update to patient specific profile

Fixed in commit https://github.com/cerner/fhir.cerner.com/pull/874/commits/6891a9e92a9faf1db82a825354e912180f12c6af

File: content/millennium/r4/foundation/other/health-cards.md (hunk 4 above)
Comment (stefanioachim) [reply content/millennium/r4/foundation/other/health-cards.md]: > should be to the specific lab results data minimization profile http://hl7.org/fhir/uv/shc-vaccination/2021Sep/StructureDefinition-shc-infectious-disease-laboratory-result-observation-dm.html

Fixed in commit https://github.com/cerner/fhir.cerner.com/pull/874/commits/6891a9e92a9faf1db82a825354e912180f12c6af

**Review by fpivonka — APPROVED** [review]

**Comment by aki-mathur** [comment]
Screenshot 2022-08-26 at 12 49 59 PM

**Pull request merged**